lib/constants/ui.ts

```diff
@@ -22,3 +22,104 @@ export const UI_ENTER_FULLSCREEN = 'UI_ENTER_FULLSCREEN';
 export const UI_LEAVE_FULLSCREEN = 'UI_LEAVE_FULLSCREEN';
 export const UI_CONTEXTMENU_OPEN = 'UI_CONTEXTMENU_OPEN';
 export const UI_COMMAND_EXEC = 'UI_COMMAND_EXEC';
+
+interface UIFontSizeSetAction {
+  type: typeof UI_FONT_SIZE_SET;
+  value: number;
+}
+interface UIFontSizeIncrAction {
+  type: typeof UI_FONT_SIZE_INCR;
+}
+interface UIFontSizeDecrAction {
+  type: typeof UI_FONT_SIZE_DECR;
+}
+interface UIFontSizeResetAction {
+  type: typeof UI_FONT_SIZE_RESET;
+}
+interface UIFontSmoothingSetAction {
+  type: typeof UI_FONT_SMOOTHING_SET;
+  fontSmoothing: string;
+}
+interface UIMoveLeftAction {
+  type: typeof UI_MOVE_LEFT;
+}
+interface UIMoveRightAction {
+  type: typeof UI_MOVE_RIGHT;
+}
+interface UIMoveToAction {
+  type: typeof UI_MOVE_TO;
+}
+interface UIMoveNextPaneAction {
+  type: typeof UI_MOVE_NEXT_PANE;
+}
+interface UIMovePrevPaneAction {
+  type: typeof UI_MOVE_PREV_PANE;
+}
+interface UIShowPreferencesAction {
+  type: typeof UI_SHOW_PREFERENCES;
+}
+interface UIWindowMoveAction {
+  type: typeof UI_WINDOW_MOVE;
+}
+interface UIWindowMaximizeAction {
+  type: typeof UI_WINDOW_MAXIMIZE;
+}
+interface UIWindowUnmaximizeAction {
+  type: typeof UI_WINDOW_UNMAXIMIZE;
+}
+interface UIWindowGeometryChangedAction {
+  type: typeof UI_WINDOW_GEOMETRY_CHANGED;
+}
+interface UIOpenFileAction {
+  type: typeof UI_OPEN_FILE;
+}
+interface UIOpenSshUrlAction {
+  type: typeof UI_OPEN_SSH_URL;
+}
+interface UIOpenHamburgerMenuAction {
+  type: typeof UI_OPEN_HAMBURGER_MENU;
+}
+interface UIWindowMinimizeAction {
+  type: typeof UI_WINDOW_MINIMIZE;
+}
+interface UIWindowCloseAction {
+  type: typeof UI_WINDOW_CLOSE;
+}
+interface UIEnterFullscreenAction {
+  type: typeof UI_ENTER_FULLSCREEN;
+}
+interface UILeaveFullscreenAction {
+  type: typeof UI_LEAVE_FULLSCREEN;
+}
+interface UIContextmenuOpenAction {
+  type: typeof UI_CONTEXTMENU_OPEN;
+}
+interface UICommandExecAction {
+  type: typeof UI_COMMAND_EXEC;
+}
+
+export type UIActions =
+  | UIFontSizeSetAction
```

Comment:
Shouldn't there be no `|` before `UIFontSizeSetAction ` ?

Reply:
It's actually inserted by Prettier, allows for easier addition/removal/rearranging of values. Similar to extra comma after the last value in a list.

Reply:
Oh okay

Reply:
It's just syntax sugar, doesn't impact code logic/execution